setup.cfg

```diff
@@ -67,7 +67,7 @@ install_requires =
     Beautifulsoup4 >= 4.0.0
     boolean.py >= 4.0
     chardet >= 3.0.0
-    click >= 6.7, !=7.0
+    click >= 6.7, !=7.0, !=8.1.8
```

Comment:
There is an issue with https://github.com/pallets/click/releases/tag/8.1.8, introduced by https://github.com/pallets/click/pull/2563/files